uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/manage-leave-requests.js

```diff
@@ -427,15 +453,16 @@ define([
 
       return {
         contact_id: prepareContactID(),
-        managed_by: vm.contactId,
+        managed_by: vm.isAdmin && filters.assignedTo !== 'me' ? undefined : vm.contactId,
```

Comment:
Add a parentheses around the condition, easier to read, `(vm.isAdmin && filters.assignedTo !== 'me')`